src/uwtools/tests/config/test_atparse_to_jinja2.py

```diff
@@ -21,7 +21,7 @@ def atparselines():
 
 @fixture
 def atparsefile(atparselines, tmp_path):
-    path = tmp_path / "atparse"
+    path = tmp_path / "atparse.txt"
```

Comment:
Any particular reason for this addition? I'm only asking because I'm scared that you're going to say that something somewhere requires all config files to have a suffix.

If it's testing an unrecognized suffix, that's less frightening.

Reply:
Good question -- I thought about commenting this line and decided not to, but I'm glad you asked. Since I made `atparse` one of the canonical format strings in the `FORMAT` object, I wanted to make sure that, when someone (probably me) did a search across the codebase to make sure that standalone instances of these strings weren't reappearing, they wouldn't be thrown off by this case, where `atparse` isn't actually meaningful in the same sense that the strings in `FORMAT` are. I figured that if someone (me) saw `atparse.txt`, they (I) would realize, "Oh, that's just a filename and could be anything".